### PR TITLE
📚 Added looping exception for Google Sheets Update operation

### PR DESCRIPTION
**Pull request by that-one-tom**
Reported by the community: https://community.n8n.io/t/google-sheets-range-expression-with-multiple-items/11261/6

### CODE REVIEW

File: docs/getting-started/key-concepts/looping.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ There are a limited number of nodes and operations where you need to design a lo
 	* **List**: This operation executes only once, not for each incoming item.
 * [**Google Sheets**](../../nodes/nodes-library/nodes/GoogleSheets/README.md) node:
 	* **Read**: This operation will execute only once for the `Sheet` resource.
```
Comment (StarfallProjects) [inline docs/getting-started/key-concepts/looping.md:83]:
```suggestion
	* **Read**: This operation executes only once for the `Sheet` resource.
```

File: docs/getting-started/key-concepts/looping.md [hunk 2]
```diff
@@ -81,6 +81,7 @@ There are a limited number of nodes and operations where you need to design a lo
...
+	* **Update**: This operation will update multiple rows only if they are in the same range.
```
Comment (StarfallProjects) [inline docs/getting-started/key-concepts/looping.md:84]: Could we say something like:

"This operation updates multiple rows if they are in the same range, or [whatever it does if they are not - updates a single row I guess?]"

File: docs/getting-started/key-concepts/looping.md [hunk 3]
```diff
@@ -81,6 +81,7 @@ There are a limited number of nodes and operations where you need to design a lo
...
 * [**HTTP Request**](../../nodes/nodes-library/core-nodes/httpRequest/README.md) node: You must handle pagination yourself. If your API call returns paginated results you must create a loop to fetch one page at a time.
 * [**Microsoft SQL**](../../nodes/nodes-library/nodes/microsoftSql/README.md) node does not natively handle looping, so if you want the node to process all incoming items you must create a loop.
 * [**Postgres**](../../nodes/nodes-library/nodes/postgres/README.md) node will execute and iterate over all incoming items only for Postgres related functions (e.g. `pgInsert`, `pgUpdate`, `pqQuery`).
```
Comment (StarfallProjects) [inline docs/getting-started/key-concepts/looping.md:87]:
```suggestion
* [**Postgres**](../../nodes/nodes-library/nodes/postgres/README.md) node will execute and iterate over all incoming items only for Postgres related functions (for example, `pgInsert`, `pgUpdate`, `pqQuery`).
```

**Review by StarfallProjects — CHANGES_REQUESTED** [review]

**Comment by StarfallProjects** [comment]
Hey! I've requested changes on the PR. They're a bit nit-picky really, but one of my aims with the PR reviews is to start steering us towards a plainer/clearer style. Please let me know if anything doesn't make sense.

And thank you for contributing 🎉

File: docs/getting-started/key-concepts/looping.md (hunk 1 above)
Comment (that-one-tom) [reply docs/getting-started/key-concepts/looping.md:83]: Looks like this change request isn't related to this PR. Maybe best to handle this separately?

File: docs/getting-started/key-concepts/looping.md (hunk 3 above)
Comment (that-one-tom) [reply docs/getting-started/key-concepts/looping.md:87]: Looks like this change request isn't related to this PR. Maybe best to handle this separately?

File: docs/getting-started/key-concepts/looping.md (hunk 2 above)
Comment (that-one-tom) [reply docs/getting-started/key-concepts/looping.md]: Thx for your feedback on this one @StarfallProjects, clarified what's happening in https://github.com/that-one-tom/n8n-docs/commit/7efac0f80933254cd4b7417d18e79baa5332163f.

File: docs/getting-started/key-concepts/looping.md (hunk 3 above)
Comment (StarfallProjects) [reply docs/getting-started/key-concepts/looping.md:87]: I'm hoping to sneak in style changes as we do other work.

File: docs/getting-started/key-concepts/looping.md (hunk 1 above)
Comment (StarfallProjects) [reply docs/getting-started/key-concepts/looping.md:83]: I'm hoping to sneak in style changes as we do other work.

**Pull request merged**